.gitignore

```diff
@@ -1,2 +1,8 @@
 fuel/app/logs/*/*/*
 fuel/app/cache/*/*
+coverage-html/
+.env
+*.tmp
+fuel/vendor/
+fuel/app/config/*/migrations.php
```

Comment:
.DS_Store is added before changed, so it is not show in here.
Please check in https://github.com/kondo-takuto-mulodo/fuelphp_unittest/blob/phpunit/.gitignore